DEPS

```diff
@@ -21,6 +22,10 @@ deps = {
   "third_party/ethash/src": "https://github.com/chfast/ethash.git@e4a15c3d76dc09392c7efd3e30d84ee3b871e9ce",
   "third_party/bitcoin-core/src": "https://github.com/bitcoin/bitcoin.git@8105bce5b384c72cf08b25b7c5343622754e7337", # v25.0
   "third_party/argon2/src": "https://github.com/P-H-C/phc-winner-argon2.git@62358ba2123abd17fccf2a108a301d4b52c01a7c",
+  "third_party/libdmg-hfsplus": {
+    "url": "https://github.com/fanquake/libdmg-hfsplus.git@1cc791e4173da9cb0b0cc16c5a1aaa25d5eb5efa",
+    "condition": "checkout_dmg_tool",
```

Comment:
I'd keep the host/checkout_mac checks you had before (in addition to `checkout_dmg_tool`).

Reply:
Implemented in f9e3fe97c108c7903b6dbe35fe386a944419537a.